packages/plugins/plugin-markdown/src/components/MarkdownEditor.tsx

```diff
@@ -177,6 +145,37 @@ export const MarkdownEditor = ({
     [id, initialValue, formattingObserver, viewMode, themeMode, extensions, providerExtensions],
   );
 
+  // Focus the space that references the comment.
```

Comment:
move function out of scope of plugin to stand alone function, with TODO to factor out and move to react-ui-editor tuil.

Reply:
I'll make this a standalone hook 👍